tests/cli/test_cli.py

```diff
@@ -74,3 +142,23 @@ def test_cli_webserver_debug(self):
             "webserver terminated with return code {} in debug mode".format(return_code))
         p.terminate()
         p.wait()
+
+    def test_local_run(self):
+        args = create_mock_args(
+            task_id='print_the_context',
+            dag_id='example_python_operator',
+            subdir='/root/dags/example_python_operator.py',
+            interactive=True,
+            execution_date=Pendulum.parse('2018-04-27T08:39:51.298439+00:00')
```

Comment:
for tests this is ok, but you can also use `timezone.parse`

Reply:
Replaced it with `timezone.parse`